cibuildwheel/__main__.py

```diff
@@ -48,26 +48,34 @@ def strtobool(val):
 def main():
     parser = argparse.ArgumentParser(
         description='Build wheels for all the platforms.',
-        epilog=('Most options are supplied via environment variables. '
-                'See https://github.com/joerick/cibuildwheel#options for info.'))
+        epilog='''
+            Most options are supplied via environment variables.
+            See https://github.com/joerick/cibuildwheel#options for info.
+        ''')
 
     parser.add_argument('--platform',
                         choices=['auto', 'linux', 'macos', 'windows'],
                         default=os.environ.get('CIBW_PLATFORM', 'auto'),
-                        help=('Platform to build for. For "linux" you need docker running, on Mac '
-                              'or Linux. For "macos", you need a Mac machine, and note that this '
-                              'script is going to automatically install MacPython on your system, '
-                              'so don\'t run on your development machine. For "windows", you need to '
-                              'run in Windows, and it will build and test for all versions of '
-                              'Python. Default: auto.'))
+                        help='''
+                            Platform to build for. For "linux" you need docker running, on Mac
+                            or Linux. For "macos", you need a Mac machine, and note that this
+                            script is going to automatically install MacPython on your system,
+                            so don't run on your development machine. For "windows", you need to
+                            run in Windows, and it will build and test for all versions of
+                            Python. Default: auto.
+                        ''')
```

Comment:
These changes are mostly unrelated - I just realised that the way I had been writing long strings was a little crazy. (argparse reformats all text, it seems)